crop_classification/dl/timl.py

```diff
@@ -625,13 +632,13 @@ def _make_tasks(
 
 def train_timl_model(
     root,
-    classifier_vector_size: int,
-    classifier_dropout: float,
-    classifier_base_layers: int,
-    num_classification_layers: int,
     model_name: str,
     encoder_vector_sizes: Union[List[int], int],
-    encoder_dropout: float,
+    classifier_vector_size: int = HIDDEN_VECTOR_SIZE,
+    classifier_dropout: float = CLASSIFIER_DROPOUT,
+    classifier_base_layers: int = CLASSIFIER_BASE_LAYERS,
+    num_classification_layers: int = NUM_CLASSIFICATION_LAYERS,
+    encoder_dropout: float = ENCODER_DROPOUT,
```

Comment:
Decided to just use the config by default here, any thoughts about this?

Reply:
yes this is great